onepwd/src/onepwd/__init__.py

```diff
@@ -66,79 +71,77 @@ def __init__(self, secret=None, shorthand=None, bin_path="", session_timeout=30)
             self.cache_token()
         else:
             raise MissingCredentials()
+            
+        check_version = self.get_version()
+        if not(check_version.startswith('2.')):
+            raise InvalidOnePwdVersion("CLI 2 is required")
```

Comment:
maybe be more specific and return: "1Password CLI Version2 is required. To check version use: "op --version""